classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def ad_student(student)
+    students << student
+    student.classroom = self
+  end
+end
```

Comment:
Add a method that serves as the 'belongs-to' side (a student belongs to a classroom) to the student.rb file 👍🏾